app/src/main/res/layout/manage_group_loyalty_card_layout.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Seems like a one-to-one copy of the regular `loyalty_card_layout.xml`, if it can be the same just use the other layout :)